ui/components/multichain/network-list-item/network-list-item.tsx

```diff
@@ -83,7 +85,7 @@ export const NetworkListItem = ({
   const [networkOptionsMenuOpen, setNetworkOptionsMenuOpen] = useState(false);
 
   const renderButton = () => {
```

Comment:
Can we convert this into a `useCallback` hook to prevent unecessary re-renders?

Reply:
☝️ This is a nit, doesn't need to block, but this function will likely cause the component to dereference as we add more of these buttons for each network

Reply:
Thanks for th review, just pushed the change!